MichaelKintscher/qualitative-coding-app
# Enable User Changes to Encoding Table Settings

Description:
Created a new file "user_settings_dialog.py" that creates a dialog popup to allow users to change the minimum cell size, maximum cell width, and padding for table cells

Testing Steps:
1) Run application from main.py
2) Click on the settings tab in the menu and open the settings dialog 3) Click the "Change Minimum Cell Size" button and ensure no error occurs in the terminal 4) Enter 50 and 50 into the minimum cell size fields, click the "Change Minimum Cell Size" button, exit the dialog then drag the cell border left to ensure the change occurred 5) Open the dialog
6) Click the "Set Maximum Cell Width" button and ensure no error occurs in the terminal 7) Enter 150 into the maximum cell width field, click the "Set Maximum Cell Width" button, exit the dialog then drag the cell border right to ensure the change occurred 8) Enter into a text box the sentence "The quick brown fox jumps over the lazy dog" 9) Open the dialog
10) Click the "Set Padding" button and ensure no error occurs in the terminal 11) Enter 20 into the set padding field, click the "Set Padding" button, then look at the sentence cell to ensure padding occurs

Patch Type:
New Feature
